ocp_resources/grafana_data_source.py

```diff
@@ -0,0 +1,10 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class GrafanaDataSource(NamespacedResource):
```

Comment:
Empty class without to_dict implement required is not allowed.
Please introduce to_dict().